test/tornado_tests/test_motor_collection.py

```diff
@@ -283,6 +285,57 @@ async def test_async_create_encrypted_collection(self):
             self.addCleanup(self.db.drop_collection, "testing1", encrypted_fields=ef)
             self.assertEqual(exc.exception.code, 121)
 
+    @gen_test
+    async def test_async_encrypt_expression(self):
```

Comment:
This test and the `test_async_create_encrypted_collection` should also be in `tests/asyncio_tests/test_asyncio_collection.py`

Reply:
Done.